Source/WebCore/platform/SharedBuffer.cpp

```diff
@@ -121,6 +160,24 @@ Ref<SharedBuffer> FragmentedSharedBuffer::makeContiguous() const
     return SharedBuffer::create(WTFMove(combinedData));
 }
 
+auto FragmentedSharedBuffer::toIPCData() const -> IPCData
+{
+    size_t bufferSize = size();
+    if (!bufferSize)
+        return WTF::Vector<std::span<const uint8_t>>();
```

Comment:
We probably don’t need this early return. I think the code below will already do the right thing.